src/Symfony/Bundle/FrameworkBundle/Resources/config/notifier_transports.php

```diff
@@ -100,7 +100,6 @@
         'sms-biuras' => Bridge\SmsBiuras\SmsBiurasTransportFactory::class,
         'sms-factor' => Bridge\SmsFactor\SmsFactorTransportFactory::class,
         'sms-sluzba' => Bridge\SmsSluzba\SmsSluzbaTransportFactory::class,
-        'sms77' => Bridge\Sms77\Sms77TransportFactory::class,
```

Comment:
This makes the framework integration of this transport entirely unusable, see failing tests. Maybe @OskarStark knows how one should deprecate a notifier bridge?